source/numerics.tex

```diff
@@ -1121,6 +1135,49 @@
 The complex hyperbolic tangent of \tcode{x}.
 \end{itemdescr}
 
+\rSec2[complex.tuple]{Tuple interface}
+
+\indexlibrarymember{tuple_size}{complex}%
+\indexlibrarymember{tuple_element}{complex}%
```

Comment:
Feel free to reject.
```output
$ grep '\\index.*tuple_' *
grep: assets: Is a directory
containers.tex:\indexlibraryglobal{tuple_size}%
containers.tex:\indexlibraryglobal{tuple_element}%
utilities.tex:\indexlibraryglobal{tuple_size}%
utilities.tex:\indexlibraryglobal{tuple_element}%
utilities.tex:\indexlibraryglobal{tuple_cat}
utilities.tex:\indexlibrary{\idxcode{tuple_size}!in general}%
utilities.tex:\indexlibraryglobal{tuple_size}%
utilities.tex:\indexlibraryglobal{tuple_element}%
utilities.tex:\indexlibraryglobal{tuple_size}%
utilities.tex:\indexlibraryglobal{tuple_element}%
```
```suggestion
\indexlibraryglobal{tuple_size}%
\indexlibraryglobal{tuple_element}%
```

Reply:
I agree with you, but I'm following the convention of similar functions (e.g. tanh), so will leave as is. I've seen this weirdness elsewhere in the spec as well and have not understood the when and why. We should open an issue.

Reply:
The reason seems obvious, it means the index looks like:

```
tanh, 1491
    complex, 1429
    valarray, 1484
```

Instead of simply:

`tanh`, 1491, 1429, 1484

Reply:
Let's go with \indexlibraryglobal for tuple_* for consistency for now,
and let's change this globally to \indexlibrarymember in a separate change.

Reply:
Ok, done. While I agree that the additional context is helpful, calling these "members" is confusing.

Reply:
It's just the name of a macro in the source, "member" is not observable in the standard.